pkgs/servers/nosql/mongodb/mongodb.nix

```diff
@@ -11,6 +11,17 @@ with stdenv.lib;
 }@args:
 
 let
+  ifdef = if versionAtLeast version "4.2"
```

Comment:
"ifdef" is not very self-explanatory, I'd change it with something like "pythonVariant"

Reply:
yeah, good suggestion, will do, thanks!